Swift VectorBoolean/VectorBoolean/BezierExtensions/NSBezierPath_Boolean.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  NSBezierPath_Boolean.swift
+//  Swift VectorBoolean for macOS
+//
+//  Based on NSBezierPath+Boolean - Created by Andrew Finnell on 5/31/11.
```

Comment:
Is this the same guy as the infamous Andy Finnell that already has code in a lot of Snagit??

Reply:
I didn't know there was already Andy Finnell code in Snagit.  Could be the same guy, have no clue.

Reply:
Seems to be... is see his name in `PathBuilder` and `SelectionBuilder` and I found the code in his BitBucket.